tests/aws/services/kms/test_kms.py

```diff
@@ -2006,6 +2006,22 @@ def test_derive_shared_secret(self, kms_create_key, aws_client, snapshot):
             )
         snapshot.match("response-invalid-public-key", e.value.response)
 
+    @markers.aws.validated
+    @markers.snapshot.skip_snapshot_verify(paths=["$..CurrentKeyMaterialId"])
+    def test_describe_with_alias_arn(self, kms_create_key, aws_client, snapshot):
+        snapshot.add_transformer(snapshot.transform.key_value("CurrentKeyMaterialId"))
+
+        alias_name = f"alias/{short_uid()}"
+        created_key = kms_create_key(Description="test - description")
+        snapshot.match("created-key", created_key)
+
+        aws_client.kms.create_alias(TargetKeyId=created_key["KeyId"], AliasName=alias_name)
+        key_arn = created_key["Arn"]
+        alias_arn = key_arn.rsplit(":", maxsplit=1)[0] + f":{alias_name}"
```

Comment:
Instead of manually creating the alias ARN here, we should use the kms operations to retrieve it, something like: 

```suggestion
        aliases = aws_client.kms.list_aliases()["Aliases"]
        alias_entry = next(a for a in aliases if a["AliasName"] == alias_name)
        alias_arn = alias_entry["AliasArn"]
```